### DISCUSSION TITLE
Option to prioritise file quality

### DISCUSSION THREAD

**Discussion by StigtriX** [nukeop/nuclear#1042] [Q&A]
Hi!

Would it be possible to prioritise file quality when one searches for music? What I mean is that if you turn this feature on, the files with 320 kbps or higher file quality will be marked with a small, green square with a note  [🎵 ]. I found that some of the songs I added to a list were a bit too compressed for my liking (or were dance practice videos etc.), and I did not immediately find a solution to discern file quality.

Keep up the good work 👍

**Comment by nukeop** [comment]
Currently the stream with the best quality is automatically selected when multiple sources are available. You can't show the quality when adding a track because the stream is only searched for once it's in the queue.

**Reply by StigtriX** [reply]
Ah, I see. It might be my fringe case where it gets a bit complicated: some K-Pop songs are released in different versions, eg. in Korean, Japanese and English, with the English (and less desirable versions) listed first.
